features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/modules/analytics/add.jag

```diff
@@ -183,76 +190,15 @@ var addDatasourceConfig = function (configData) {
-            error: false,
-            message: message
-        };
-    }
-    if(configData.eventAnalyserUsername.trim() == ""){
-        var message = "Event Analyzer Username is empty.";
-        log.error(message);
-        return {
-            error: true,
-            message: message
-        };
-    }
-    if(configData.eventAnalyserUsername.trim().indexOf(" ") > -1){
-        var message = "Event Analyzer Username cannot contain space character.";
-        log.error(message);
-        return {
-            error: true,
-            message: message
-        };
-    }
-    if(configData.eventAnalyserPassword.trim() == ""){
-        var message = "Data Analyzer Password is empty.";
-        log.error(message);
-        return {
-            error: true,
-            message: message
-        };
-    }
-    try {
-        var process = require('process');
-        var cAppPath = process.getProperty('carbon.home') + '/statistics';
-        var cAppName = '';
-
-        if (configData.statDsDriver.toLowerCase().indexOf('oracle') !=-1) {
-            cAppName = 'API_Manager_Analytics_Oracle.car';
-        } else {
-            cAppName = 'API_Manager_Analytics.car';
-        }
-        cAppPath = cAppPath + '/' + cAppName;
-        var file = new java.io.File(cAppPath);
 
-        var byteArray = org.apache.commons.io.FileUtils.readFileToByteArray(file);
-        var dataHandler = new javax.activation.DataHandler(byteArray, "application/octet-stream");
-
-        var stub = new org.wso2.carbon.application.mgt.stub.upload.CarbonAppUploaderStub(configData.
-                eventAnalyserURL + '/services/CarbonAppUploader');
-        var client = stub._getServiceClient();
-        var options = client.getOptions();
-        var authenticator = new org.apache.axis2.transport.http.HttpTransportProperties.Authenticator();
-        authenticator.setUsername(configData.eventAnalyserUsername);
-        authenticator.setPassword(configData.eventAnalyserPassword);
-        authenticator.setPreemptiveAuthentication(true);
-        options.setProperty(org.apache.axis2.transport.http.HTTPConstants.AUTHENTICATE, authenticator);
-        client.setOptions(options);
-        log.info("Deploying DAS cApp '" + cAppName + "'...");
-        var fileItem = new org.wso2.carbon.application.mgt.stub.upload.types.carbon.UploadedFileItem();
-        fileItem.setDataHandler(dataHandler);
-        fileItem.setFileName(cAppName);
-        fileItem.setFileType("jar");
-        stub.uploadApp([fileItem]);
-        log.info("DAS cApp '" + cAppName + "' deployed");
+    try {
+        var client=org.wso2.carbon.apimgt.usage.client.UsageClient.getClient("");
+        client.deployArtifacts(configData.eventAnalyserURL,configData.eventAnalyserUsername,configData.eventAnalyserPassword);
         return {
             error: false
         };
-    } catch (ex) {
-        var message = "Error occurred while installing BAM Toolbox (Cause:" + ex.message + ")";
+    }catch(ex){
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea